json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -195,43 +195,48 @@ protected function get_plugin_updates( $plugin_file ) {
 	 * Ref: https://codex.wordpress.org/Plugin_API/Filter_Reference/plugin_action_links_(plugin_file_name)
 	 * @return array of plugin action links (key: link name value: url)
 	 */
-	 protected function get_plugin_action_links( $plugin_file ) {
```

Comment:
Sorry for all the red; there were some extra spaces on each line and my editor stripped them on save.